### PR TITLE
Fixes #9159 - Fix rubocop errors

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

**Comment by domcleal** [comment]
http://ci.theforeman.org/job/test_plugin_pull_request/492/ is running.

File: .rubocop.yml [hunk 1]
```diff
@@ -24,3 +24,6 @@ StringLiterals:
 
 Metrics/ClassLength:
   Max: 200
+
+Metrics/AbcSize:
+  Enabled: false # should we turn this on?
```
Comment (eLobato) [inline .rubocop.yml]: I think so, in any case I'll merge this and we can enable it later.

Comment (daviddavis) [inline .rubocop.yml]: Yea, there were several failures and these types of cops are never easy to fix.

**Comment by daviddavis** [comment]
@eLobato removed rubocop from the gemspec.

**Comment by eLobato** [comment]
@daviddavis http://ci.theforeman.org/job/test_plugin_pull_request/494/ :white_check_mark: , merging..

**Comment by eLobato** [comment]
Merged as ac230210a5d69c663012c45b6531c4f2bb1dc3ff
 , thanks @daviddavis

**Pull request closed without merging**